index.html

```diff
@@ -52,15 +48,186 @@ <h2 class="main_section_h2">I'm a software developer</h2>
           <a class="social_media_icon" id="medium_icon">
             <img src="assets/medium.svg" alt="medium icon "/>
           </a>
+        </li>
+         <li>
+          <a class="social_media_icon">
+            <img src="assets/github.svg" alt="github icon "/>
+          </a>
         </li>
         <li>
-          <a class="social_media_icon" >
-            <img src="assets/twitter.svg" alt="twitter icon "/>
+          <a class="social_media_icon">
+            <img src="assets/angellist.svg" alt="angellist icon "/>
           </a>
         </li>
       </ul>
+      <div class="next_page_icon">
+        <a href=".work_section">
+          <svg width="42" height="42" viewBox="0 0 42 42" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <rect x="1" y="1" width="40" height="40" rx="20" fill="white"></rect>
+            <path fill-rule="evenodd" clip-rule="evenodd" d="M14.2929 17.2929C14.6834 16.9024 15.3166 16.9024 15.7071 17.2929L21 22.5858L26.2929 17.2929C26.6834 16.9024 27.3166 16.9024 27.7071 17.2929C28.0976 17.6834 28.0976 18.3166 27.7071 18.7071L21.7071 24.7071C21.3166 25.0976 20.6834 25.0976 20.2929 24.7071L14.2929 18.7071C13.9024 18.3166 13.9024 17.6834 14.2929 17.2929Z" fill="#172B4D"></path>
+            <rect x="1" y="1" width="40" height="40" rx="20" stroke="#DFE1E6"></rect>
+          </svg>
+        </a>
+      </div>
 
     </section>
+    <section class="work_section">
+     
+      <h1 class="work_section_title">My Recent <br> Works</h1>
+      <div class="delimiter">
+
+      </div>
+      <div class="work_section_content">
+        <div class="work_element">
+          <div class="work_image">
+
+          </div>
+          <div class="work_details">
+            <h3 class="work_title">
+              Multi-Post Stories <br> Gain+Glory
+            </h3>
+            <ul class="work_languages">
+              <li class="work_language_item">Ruby on rail</li>
+              <li class="work_language_item">css</li>
+              <li class="work_language_item">JavaScript</li>
+              <li class="work_language_item">html</li>
+            </ul>
+            <button class="work_more_detail_button"> See Project</button>
+          </div>
+        </div>
+        <div class="work_element">
+          <div class="work_image">
```

Comment:
Required

- Great work with the rest of the design it looks neat. However, please replace this dev with an actual image to follow the semantics and the design

Reply:
@MahmoudBakr23 , thank you for your remarks, please there is no image there there is no image in the template . I'm using the template of the figma design. please can I use the images of template 1?